### PR TITLE
Allow .check() to be alias for .solution() in ThoughtExperiments

### PR DESCRIPTION
**Pull request by dansbecker**

### CODE REVIEW

**Comment by dansbecker** [comment]
@alexisbcook Here's the PR we talked about. 
🤞

**Comment by dansbecker** [comment]
@skbergam @alexisbcook 

Alexis wanted to start using `check()` instead of `solution()` for Thought Questions (where we don't check their code and instead just show them the answer), because it's more obvious to users that they need to run it in order to get credit for the course.

I created this PR so that the .check method shows the solution content, and then records the question as correct. 

These questions will now show up as having a correct call to .check rather than just a call to "solution." It's admittedly hacky, but we've tested it and it does work for advancing user's progress when they call the .check() method on a Thought Question.

Hoping to get Sean's view on how concerned he is with this approach (recognizing that the code cleanliness standards for learntools have historically been lower than for the core Kaggle codebase) before we merge this.

File: learntools/core/problem.py [hunk 1]
```diff
@@ -68,17 +68,9 @@ def check_whether_attempted(self, *args):
 
 class ThoughtExperiment(Problem):
 
-    # By default, ThoughtExperiment subclasses have no bearing on progress tracking.
-    _counts_for_points = False
```
Comment (dansbecker) [inline learntools/core/problem.py]: The _counts_for_points property no longer has been deprecated, and this line should be removed no matter what. It has no effect any more.

File: learntools/core/problem.py [hunk 2]
```diff
@@ -68,17 +68,9 @@ def check_whether_attempted(self, *args):
...
-    
+    show_solution_on_correct = True
```
Comment (dansbecker) [inline learntools/core/problem.py]: Causes solution to be shown

File: learntools/core/problem.py [hunk 3]
```diff
@@ -68,17 +68,9 @@ def check_whether_attempted(self, *args):
...
     def check(self, *args):
-        # TODO: Would be nice to be able to put the variable name this problem is
-        # bound to here. (If we want to do that, it should probably live up one
-        # level in the ProblemView?)
-        msg = ("Nothing to check! (Just do this one in your head, then"
-                " call `.solution()` to see if your prediction was correct.)")
-        raise Uncheckable(msg)
-
+        pass
```
Comment (dansbecker) [inline learntools/core/problem.py]: Check mehtods count a question as Correct if no assertions fail. 

By not having assertions, there's nothing to fail. This method will always report that the user got the correct answer.

**Comment by skbergam** [comment]
On it's face this looks gtg. Looks like it should have no effect on other types of questions given they all inherit from `Problem` (at least that's what I assume from looking at this code) so if it's been tested then LGTM

**Pull request merged**